test/release/examples/primers/fact.c

```diff
@@ -0,0 +1,40 @@
+
+#include<stdio.h>
+#include<stdlib.h>
+#include"fact.h"
+
+//#include<arrow-glib/arrow-glib.h>
```

Comment:
Good catch! I am trying to remove all Apache Arrow/Parquet references to avoid confusing anyone who is not familiar. I just had them in my code when I was trying a few things out.